vmdb/app/helpers/container_group_helper/textual_summary.rb

```diff
@@ -41,6 +41,10 @@ def textual_dns_policy
     {:label => "DNS Policy", :value => @record.dns_policy}
   end
 
+  def textual_ip
+    {:label => "IP", :value => @record.ip}
```

Comment:
> I'm curious, is there a hostname field? Do pods support IPv6?

@jrafanie ,
So far k8s does not support IPv6 (please see [here](https://github.com/ManageIQ/manageiq/pull/2828#discussion_r29555179))
There's no hostname field. There's a host field in the spec section where the user *can express*
which is the preferred host to be ran on, but the runtime *actual* assignment is expressed by this IP field only.
Please see [this](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/v1beta3/types.go#L820) and [this](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/api/v1beta3/types.go#L835) 

@zeari , 
Please address the comment above to expedite final review and merge: 
https://github.com/ManageIQ/manageiq/pull/2828#discussion_r29555179

cc @simon3z